src/api/script/api_test_in_spec.sh

```diff
@@ -63,15 +63,12 @@ EOF
 /usr/sbin/memcached $MEMCACHED_USER -l 127.0.0.1 -d -P $MEMCACHED_PID_FILE || exit 1
 
 # migration test
```

Comment:
The 001 migration doesn't run in this scenario. The dump is also filling some schema versions into the corresponding table:

```
INSERT INTO `schema_migrations` VALUES ('1'),('10'),('11'),('12'),('13'),('14'),('15'),('16'),('17'),('18'),('19'),('2'),('20'),('20090701125033'),('20090703100900'),('20090716174522'),
```

The only difference I saw on my instance (note I was using the same DB name and default DB version we have in our vagrant box), was the change of the indexing method.
Which I think we should change via a migration instead of hiding the difference, if possible.